src/JuMP.jl

```diff
@@ -199,6 +199,8 @@ end
 @Base.deprecate getNumVars(m::Model) MathProgBase.numvar(m)
 @Base.deprecate getNumConstraints(m::Model) MathProgBase.numlinconstr(m)
 
+getCost(m::Model) = m.redCosts
```

Comment:
Not exactly. The getDual returns the costs of one variable, whereas the redCosts returns the costs associated with the entire model.